Controllers/Roles Auth Controllers/superAdminAuthController.js

```
@@ -0,0 +1,73 @@
+        role: "superadmin",
+      });
+
+      // Return the token
+      res.status(200).json(result);
+    } catch (error) {
+      res
+        .status(400)
+        .json({ error: `SuperAdminAuthController-Login: ${error.message}` });
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)